bento/src/test/java/com/yelp/android/bento/components/ListComponentTest.java

```diff
@@ -141,6 +144,68 @@ public int getSpanSize(int position) {
         assertEquals(10, mListComponent.getSpanSizeLookup().getSpanSize(0));
     }
 
+    @Test
+    public void test_WhenDividerEnabledAndOnItemVisibleCalled_CallsOnListItemVisibleForCorrectIndices() {
```

Comment:
Can we remove all the "test_" from this class? We are on JUnit4 now so they are not necessary.

Reply:
Done!